doc/api/fs.md

```diff
@@ -891,18 +884,10 @@ changes:
 
 Synchronously tests a user's permissions for the file or directory specified by
 `path`. The `mode` argument is an optional integer that specifies the
-accessibility checks to be performed. The following constants define the
+accessibility checks to be performed. Check [`File Access Contants`](#fs_file_access_constants) for
```

Comment:
1. Contants -> Constants
2. This line needs to be wrapped at 80 characters. Or you can create a bottom reference and use `` [`File Access Constants`][] ``.